### ISSUE TITLE
PxP - Naming conventions for model output files

### ISSUE THREAD

**Issue by andkov** [IALSA/IALSA-2015-Portland#72]

**Comment by andkov** [comment]
@annierobi, i've uploaded a trail model [b1_female_aehplus_grip_pek](https://github.com/IALSA/IALSA-2015-Portland/blob/master/projects/physical/outputs/b1_female_aehplus_grip_pek.out).

Please note that the renaming occurred in the [DEFINE statement](https://github.com/IALSA/IALSA-2015-Portland/blob/master/projects/physical/outputs/b1_female_aehplus_grip_pek.inp#L37)
```
Define: p1=gripp1; p2=gripp2; p3=gripp3; p4=gripp4; p5=gripp5;
        c1=pek1; c2=pek2; c3=pek3; c4=pek4; c5=pek5;
```
leaving the elements in the MODEL statement unaltered:
```
MODEL:  ip sp | p1-p5 AT time1-time5;
        ic sc | c1-c5 AT time1-time5;
```
at this point it would be useful to thing about "p" and "c" as indicators of the first and the second processes respectively.  We could have re-named it to be "process A" and "process B", and we'll do so in the future, but for now "process A" will be identified by a letter "p" and "process B" will be identified by a letter "c". This will require the least amount of script adjustment, both in Mplus specification and in R extraction.

**Comment by annierobi** [comment]
@andkov, I had used p for pek and c for gripp:) I am assuming we need to use the same naming. I will make the change and upload the file. Whatever naming convention we choose, we need to make sure everyone knows which name should be used for each variable (e.g. c = pek and p = gripp)

**Comment by andkov** [comment]
@annierobi, yes. Please refer to this issue thread as the authority on the naming decisions when you correspond with others participants. You can add the link in the comment window by using markdown language: use 
```
[ text label ](http://...)
```
So type in
```
See  [naming convention](https://github.com/IALSA/IALSA-2015-Portland/issues/72)
```
into this comment window order to create a link.  Like this:  
See [naming convention](https://github.com/IALSA/IALSA-2015-Portland/issues/72)
